src/components/input-number/input-number.tsx

```diff
@@ -131,9 +131,9 @@ export class InputNumber
   @Prop({ reflect: true }) hidden = false;
 
   /**
-   * When `true`, shows a default recommended icon. Alternatively, pass a Calcite UI Icon name to display a specific icon.
+   * Pass a Calcite UI Icon name to display a specific icon.
```

Comment:
WDYT about the following (in line with other components such as `action` and `chip`:

> Specifies an icon to display.